app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/acquisitiondetails/check_your_answers.scala.html

```diff
@@ -30,7 +32,7 @@
 returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link
 )
 
-@(answers: CompleteAcquisitionDetailsAnswers)(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
+@(answers: CompleteAcquisitionDetailsAnswers, cutoffDate: LocalDate)(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
```

Comment:
`cutoffDate` is a bit vague here - it'd be easier to know what it actually is with a clearer name